### PR TITLE
Doc Update: P Scaling in Sasmodels

### PR DESCRIPTION
**Pull request by smk78**
This PR is one of two addressing https://github.com/SasView/sasview/issues/1708. The other being https://github.com/SasView/sasview/pull/1956.

It adds a note at the foot of the Theory section in fitting_sq.rst

### CODE REVIEW

File: doc/guide/fitting_sq.rst [hunk 1]
```diff
@@ -283,6 +283,11 @@ and the form:shell volume ratio as
 
     \phi_S = \phi_P \langle V_e \rangle \big/ \langle V_s \rangle
 
+.. note::
+
+    A change to how the shell volume is incorporated in the overall volume
+    normalisation was made in Sasmodels v1.0.5 in Nov 2020. The technical
+    details can be found `here <https://github.com/SasView/sasview/issues/1698#issuecomment-718979190>`_.
```
Comment (pkienzle) [inline doc/guide/fitting_sq.rst:290]: The "technical details" I provide in the link are too terse to be useful. How about:
```rst
Prior to Sasmodels v1.0.5 (Nov 2020), the intermediated $P(Q)$ returned by
the interaction calculator did not incorporate the volume normalization and
so $I(Q) \ne P(Q) S(Q)$. This was apparent when $P(Q)$ and $I(Q)$ were plotted together.
```

Comment (smk78) [reply doc/guide/fitting_sq.rst:290]: Have changed the note to read:

image

'here' now just resolves to https://github.com/SasView/sasview/issues/1698

**Comment by smk78** [comment]
Have also removed the 'This help document is under development' note at the start of fitting_sq.rst

**Review by pkienzle — APPROVED** [review]

**Comment by wpotrzebowski** [comment]
sasmodels ready for testing on OSX

**Pull request merged**